src/messaging/tests/TestReliableMessageProtocol.cpp

```diff
@@ -225,92 +225,97 @@ struct BackoffComplianceTestVector
-    },
-    {
-        .sendCount   = 1,
-        .backoffBase = System::Clock::Timeout(4000),
-        .backoffMin  = System::Clock::Timeout(4400),
-        .backoffMax  = System::Clock::Timeout(5500),
-    },
-    {
-        .sendCount   = 2,
-        .backoffBase = System::Clock::Timeout(4000),
-        .backoffMin  = System::Clock::Timeout(7040),
-        .backoffMax  = System::Clock::Timeout(8800),
-    },
-    {
-        .sendCount   = 3,
-        .backoffBase = System::Clock::Timeout(4000),
-        .backoffMin  = System::Clock::Timeout(11264),
-        .backoffMax  = System::Clock::Timeout(14081),
-    },
-    {
-        .sendCount   = 4,
-        .backoffBase = System::Clock::Timeout(4000),
-        .backoffMin  = System::Clock::Timeout(18022),
-        .backoffMax  = System::Clock::Timeout(22529),
-    },
-    {
-        .sendCount   = 5,
-        .backoffBase = System::Clock::Timeout(4000),
-        .backoffMin  = System::Clock::Timeout(28835),
-        .backoffMax  = System::Clock::Timeout(36045),
-    },
-    {
-        .sendCount   = 6,
-        .backoffBase = System::Clock::Timeout(4000),
-        .backoffMin  = System::Clock::Timeout(28835),
-        .backoffMax  = System::Clock::Timeout(36045),
-    },
-};
+struct BackoffComplianceTestVector theBackoffComplianceTestVector[] = { {
+                                                                            .sendCount   = 0,
+                                                                            .backoffBase = System::Clock::Timeout(300),
+                                                                            .backoffMin  = System::Clock::Timeout(330),
+                                                                            .backoffMax  = System::Clock::Timeout(413),
+                                                                        },
+                                                                        {
+                                                                            .sendCount   = 1,
+                                                                            .backoffBase = System::Clock::Timeout(300),
+                                                                            .backoffMin  = System::Clock::Timeout(330),
+                                                                            .backoffMax  = System::Clock::Timeout(413),
+                                                                        },
+                                                                        {
+                                                                            .sendCount   = 2,
+                                                                            .backoffBase = System::Clock::Timeout(300),
+                                                                            .backoffMin  = System::Clock::Timeout(528),
+                                                                            .backoffMax  = System::Clock::Timeout(661),
```

Comment:
Why did these change with off-by-one?

Reply:
Because I realized that some maximums were not correct given that we approximate 1.1 with 1127/1024 for faster arithmetics.